sarathi/til.py

```diff
@@ -385,6 +386,27 @@ def fix_urls():
                     sys.stderr.write(f"Failed getting title for `{url}`")
     write_til_json(til_json)
 
+def print_fortune(args=None):
```

Comment:
See comments above about naming the function.

Additionally, this function can exist in its own module. So make a new file, call it `fortune.py`, and add this function there instead. Remember to run your tests and check that they work before you push the code.

Reply:
Another good practice is to add a function docstring whenever you make a function. Get into the habit of writing the documentation first and then writing the function, so you will have better clarity into why you're writing a function.

```python

def print_fortune(args=None):
    """Returns the output of the Linux `fortune` comment.
    """"
    ...

```